doc/config-generic-update-rollback/Json_Patch_Ordering_using_YANG_Models_Design.md

```diff
@@ -0,0 +1,698 @@
+        "192.0.0.4": {}
+    }
+}
+```
+**JsonPatch**
+```json
+[
+    {
+        "path": "/DHCP_SERVER/192.0.0.5",
+        "op": "add",
+        "value": {}
+    }
+]
+```
+
+**Result Jsonchanges**
+```json
+[
+    [{"op": "add", "path": "/DHCP_SERVER/192.0.0.5", "value": {}}]
+]
+```
+
+## A.2 Dynamic Port Breakout 1 to 4
+**Current running config**
+```json
+{
+    "PORT": {
+        "Ethernet0": {
+            "alias": "Eth1",
+            "lanes": "65, 66, 67, 68",
+            "description": "Ethernet0 100G link",
+            "speed": "100000"
+        }
+    },
+    "ACL_TABLE": {
+        "NO-NSW-PACL-V4": {
+            "type": "L3",
+            "policy_desc": "NO-NSW-PACL-V4",
+            "ports": [
+                "Ethernet0"
+            ]
+        }
+    },
+    "VLAN_MEMBER": {
+        "Vlan100|Ethernet0": {
+            "tagging_mode": "untagged"
+        }
+    }
+}
+```
+**JsonPatch**
+```json
+[
+    {
+        "op": "add",
+        "path": "/PORT/Ethernet3",
+        "value": {
+            "alias": "Eth1/4",
+            "lanes": "68",
+            "description": "",
+            "speed": "10000"
+        }
+    },
+    {
+        "op": "add",
+        "path": "/PORT/Ethernet1",
+        "value": {
+            "alias": "Eth1/2",
+            "lanes": "66",
+            "description": "",
+            "speed": "10000"
+        }
+    },
+    {
+        "op": "add",
+        "path": "/PORT/Ethernet2",
+        "value": {
+            "alias": "Eth1/3",
+            "lanes": "67",
+            "description": "",
+            "speed": "10000"
+        }
+    },
+    {
+        "op": "replace",
+        "path": "/PORT/Ethernet0/lanes",
+        "value": "65"
+    },
+    {
+        "op": "replace",
+        "path": "/PORT/Ethernet0/alias",
+        "value": "Eth1/1"
+    },
+    {
+        "op": "replace",
+        "path": "/PORT/Ethernet0/description",
+        "value": ""
+    },
+    {
+        "op": "replace",
+        "path": "/PORT/Ethernet0/speed",
+        "value": "10000"
+    },
+    {
+        "op": "add",
+        "path": "/VLAN_MEMBER/Vlan100|Ethernet2",
+        "value": {
+            "tagging_mode": "untagged"
+        }
+    },
+    {
+        "op": "add",
+        "path": "/VLAN_MEMBER/Vlan100|Ethernet3",
+        "value": {
+            "tagging_mode": "untagged"
+        }
+    },
+    {
+        "op": "add",
+        "path": "/VLAN_MEMBER/Vlan100|Ethernet1",
+        "value": {
+            "tagging_mode": "untagged"
+        }
+    },
+    {
+        "op": "add",
+        "path": "/ACL_TABLE/NO-NSW-PACL-V4/ports/1",
+        "value": "Ethernet1"
+    },
+    {
+        "op": "add",
+        "path": "/ACL_TABLE/NO-NSW-PACL-V4/ports/2",
+        "value": "Ethernet2"
+    },
+    {
+        "op": "add",
+        "path": "/ACL_TABLE/NO-NSW-PACL-V4/ports/3",
+        "value": "Ethernet3"
+    }
+]
+```
+
+**Result Jsonchanges**
+```json
+[
+    [{"op": "remove", "path": "/ACL_TABLE/NO-NSW-PACL-V4/ports"}],
```

Comment:
In this case, we are modifying the port field from Ethernet0 to empty, but we are not removing it right?

Reply:
It will be removed.

If we are to make it empty the jsonpatch would be:
    [{"op": "replace", "path": "/ACL_TABLE/NO-NSW-PACL-V4/ports", "value":""}],